### PR TITLE
Client Registry Task Handling - update back end

### PR DESCRIPTION
**Pull request by rayking29**
# Fixes or Implements [AB#13741](https://dev.azure.com/qslvic/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/13741)

## Description

Client Registry Task Handling - update back end to allow the response code to be returned with the PatientModel if in the list of codes, rather than an error.

## Testing

- [ ] Unit Tests Updated
- [ ] Functional Tests Updated
- [x] Not Required

## Items to Review:

-   [General PR Guidelines](https://github.com/bcgov/healthgateway/wiki/PRguidance)

### CODE REVIEW

File: Testing/functional/tests/.eslintrc.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
     "env": {
-        "cypress/globals": true
+        "cypress/globals": true,
+        "es2021": true
```
Comment (BrianMaki) [inline Testing/functional/tests/.eslintrc.json:4]: What's this change for?

**Review by BrianMaki — COMMENTED** [review]
In the PR it says tests are not required but this PR has functional test changes.

File: Apps/Admin/Tests/Functional/.eslintrc.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
     "env": {
-        "cypress/globals": true
+        "cypress/globals": true,
+        "es2021": true
```
Comment (BrianMaki) [inline Apps/Admin/Tests/Functional/.eslintrc.json:4]: Why is there an Admin change for functional tests?  And what is this change for?

File: Testing/functional/tests/.eslintrc.json (hunk 1 above)
Comment (MikeLyttle) [reply Testing/functional/tests/.eslintrc.json:4]: That came from a commit from dev.  This PR should be updated to drop all the commits except the last one.

File: Apps/AdminWebClient/src/ClientApp/package.json [hunk 3]
```diff
@@ -14,19 +14,19 @@
         "@fortawesome/free-solid-svg-icons": "^6.1.2",
         "@fortawesome/vue-fontawesome": "^2.0.8",
         "@types/file-saver": "^2.0.5",
-        "core-js": "^3.24.1",
+        "core-js": "^3.25.0",
```
Comment (BrianMaki) [inline Apps/AdminWebClient/src/ClientApp/package.json:17]: What is admin web client's package-lock and package.json files changed?

File: Apps/Common/src/AspNetConfiguration/Modules/Audit.cs [hunk 4]
```diff
@@ -33,13 +35,25 @@ public static class Audit
         /// </summary>
         /// <param name="services">The services collection provider.</param>
         /// <param name="logger">The logger to use.</param>
-        public static void ConfigureAuditServices(IServiceCollection services, ILogger logger)
```
Comment (BrianMaki) [inline Apps/Common/src/AspNetConfiguration/Modules/Audit.cs:36]: Are these changes in your Audit part of your changes?

**Review by BrianMaki — CHANGES_REQUESTED** [review]
Can you please review your changes and only include those that are part of your task?

**Pull request closed without merging**